service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -379,58 +376,63 @@ private AddressExportDetailsDto getAddressDtoForAdminPage(Address address) {
      * status with some translation with some language and arrays with orderStatuses
      * and OrderPaymentStatuses with translation.
      * 
-     * @param order    {@link Order}.
-     * @param language {@link Language}.
+     * @param order {@link Order}.
      * @return {@link GeneralOrderInfo}.
      *
      * @author Yuriy Bahlay.
      */
-    private GeneralOrderInfo getInfoAboutStatusesAndDateFormed(Optional<Order> order, Language language) {
+    private GeneralOrderInfo getInfoAboutStatusesAndDateFormed(Optional<Order> order) {
         OrderStatus orderStatus = order.isPresent() ? order.get().getOrderStatus() : OrderStatus.CANCELED;
         Optional<OrderStatusTranslation> orderStatusTranslation =
-            orderStatusTranslationRepository.getOrderStatusTranslationByIdAndLanguageId(orderStatus.getNumValue(),
-                languageRepository.findIdByCode(language.getCode()));
+            orderStatusTranslationRepository.getOrderStatusTranslationById(orderStatus.getNumValue());
         String currentOrderStatusTranslation =
             orderStatusTranslation.isPresent() ? orderStatusTranslation.get().getName() : "order status not found";
-        String currentOrderStatusPaymentTranslation = null;
+        String currentOrderStatusTranslationEng =
+            orderStatusTranslation.isPresent() ? orderStatusTranslation.get().getNameEng()
+                : "order status english not found";
```

Comment:
this is wrong. I'd better return enum value (OrderStatus) than this message. It's confusing for user